common/taskana-common/src/main/java/pro/taskana/common/api/LocalTimeInterval.java

```diff
@@ -1,35 +1,37 @@
 package pro.taskana.common.api;
 
 import java.time.LocalTime;
+import java.util.Objects;
 
-public class LocalTimeInterval {
+/** Note: this class has a natural ordering that is inconsistent with equals. */
```

Comment:
Create JavaDoc comment according to our guidelines.
See: https://taskana.atlassian.net/wiki/spaces/TAS/pages/2043609182/JavaDoc+Rules